### PR TITLE
added link to docs

### PR DESCRIPTION
**Pull request by Fl4m3Ph03n1x**
Adding more information on where to go for the ssl_options flag. 
Follow up from this forum discussion:

https://elixirforum.com/t/how-to-connect-to-postgrex-with-ssl-certificate/26255

### CODE REVIEW

File: lib/postgrex.ex [hunk 1]
```diff
@@ -79,7 +79,8 @@ defmodule Postgrex do
     * `:handshake_timeout` - Connection handshake timeout in milliseconds
       (defaults to `:timeout` value);
     * `:ssl` - Set to `true` if ssl should be used (default: `false`);
-    * `:ssl_opts` - A list of ssl options, see ssl docs;
+    * `:ssl_opts` - A list of ssl options, see the
+    [ssl docs](http://erlang.org/doc/man/ssl.html);
```
Comment (wojtekmach) [inline lib/postgrex.ex]: good call. How about linking to particular function:

```markdown
* `:ssl_opts` - A list of ssl options, see `:ssl.connect/2`;
```

?

Comment (Fl4m3Ph03n1x) [reply lib/postgrex.ex]: I agree that more precise documentation would be more helpful, it's just that as this is my first PR into this project I was unsure if my suggestions was too specific.

From my understanding, the connect options the `ssl_opts` expects is actually [`tls_client_option`](http://erlang.org/doc/man/ssl.html#type-tls_client_option). 

Would this be accurate?

Comment (wojtekmach) [reply lib/postgrex.ex]: Yup, let's go with that :+1:.

**Comment by hauleth** [comment]
Partially this was done in the #485, but I agree, that some more information in docs would be helpful for newcomers.

👍

**Comment by wojtekmach** [comment]
@hauleth oh, nice, I missed that pr. @Fl4m3Ph03n1x could use also change that type to use tls_client_option as its more specific?

**Comment by Fl4m3Ph03n1x** [comment]
I did some modifications, please let me know if I missed something !

**Comment by wojtekmach** [comment]
Thank you!

**Pull request merged**